crypto/info.c

```diff
@@ -36,6 +77,11 @@ const char *OPENSSL_info(int t)
             static const char list_sep[] = { LIST_SEPARATOR_CHAR, '\0' };
             return list_sep;
         }
+    case OPENSSL_INFO_CPU_SETTINGS:
+        if (RUN_ONCE(&ossl_init_info_once, do_init_info)
```

Comment:
Since this is generic, I propose that this RUN_ONCE is run at the top of this function, before the switch.  There's no need to care about its return value...

Reply:
ah, yes. good point.